experimental/PropertyDDS/packages/property-inspector-table/src/DecoratedSelect.tsx

```diff
@@ -10,96 +22,82 @@ import {
-
-import {SvgIcon} from './SVGIcon';
-
-import { makeStyles } from '@material-ui/core/styles';
-import classNames from 'classnames';
-import React, { useState } from 'react';
-import Select from 'react-select';
-import { IndicatorProps } from 'react-select/lib/components/indicators';
-import Option, { OptionProps } from 'react-select/lib/components/Option';
-import SingleValue, { SingleValueProps } from 'react-select/lib/components/SingleValue';
-import { Props as SelectProps } from 'react-select/lib/Select';
-import { StylesConfig } from 'react-select/lib/styles';
-import { GroupedOptionsType, OptionsType, ValueType } from 'react-select/lib/types';
+} from "./constants";
 
 const useStyles = makeStyles({
 
   customOptionLabel: {
-    marginTop: '4px',
+    marginTop: "4px",
   },
   dropdown: {
-    borderRadius: '4px',
-    marginBottom: '5px',
+    borderRadius: "4px",
+    marginBottom: "5px",
   },
   dropdownClosed: {
-    border: '2px solid ' + backGroundGrayColor,
+    border: `2px solid ${ backGroundGrayColor }`,
   },
   dropdownOpen: {
-    border: '2px solid ' + borderBlueColor,
+    border: `2px solid ${ borderBlueColor }`,
   },
   menuIndicatorIcon: {
-    height: '16px',
-    marginRight: '8px',
-    width: '16px',
+    height: "16px",
+    marginRight: "8px",
+    width: "16px",
   },
-}, { name: 'DecoratedSelect' });
+}, { name: "DecoratedSelect" });
 
 // Style api of react-select library
 const reactSelectStyles: StylesConfig = {
   container: (provided) => ({
     ...provided,
-    backgroundColor: 'transparent',
+    backgroundColor: "transparent",
   }),
   control: (base) => ({
     ...base,
-    backgroundColor: backGroundDarkGrayColor + ' !important',
-    border: 'none !important',
-    boxShadow: '1px 1px 10px 1px ' + transparentShadowColor + ' !important',
+    backgroundColor: `${ backGroundDarkGrayColor } !important`,
+    border: "none !important",
+    boxShadow: `1px 1px 10px 1px ${ transparentShadowColor } !important`,
   }),
   indicatorSeparator: (provided, state) => ({
     ...provided,
-    display: 'none',
+    display: "none",
   }),
   indicatorsContainer: (provided) => ({
     ...provided,
     background: backGroundDarkGrayColor,
-    borderRadius: '5px',
+    borderRadius: "5px",
   }),
   menu: (base) => ({
     ...base,
-    marginBottom: '0px',
-    marginTop: '5px',
+    marginBottom: "0px",
+    marginTop: "5px",
   }),
-  option: (provided, state) => (Object.assign({
+  option: (provided, state) => ({
     ...provided,
     color: textDarkColor,
-    display: 'flex',
-    fontSize: '.9rem',
-    fontWeight: 'bold',
-  },
-    state.isFocused ? { backgroundColor: backGroundLightBlueColor } : {},
-    !state.isFocused ? { backgroundColor: colorWhite } : {},
-  )),
+    display: "flex",
+    fontSize: ".9rem",
+    fontWeight: "bold",
+    ...(state.isFocused ? { backgroundColor: backGroundLightBlueColor } : {}),
+    ...(!state.isFocused ? { backgroundColor: colorWhite } : {}),
+  }),
```

Comment:
Interesting trick to leave the property unset (vs. explicitly set to the value undefined).

Reply:
Honestly here it should probably just be

```ts
backgroundColor: state.isFocused ? backGroundLightBlueColor : colorWhite,
```

But I just took the auto-fix.